modules/bpi/bpi.images.inc

```diff
@@ -26,7 +26,8 @@ function bpi_syndicate_images($type = 'ajax') {
   $commands[] = ajax_command_ding_popup(
     'bpi-syndicate-images',
     t('Syndicate images'),
-    drupal_render($form)
+    drupal_render($form),
+    array('refresh' => TRUE)
```

Comment:
We need the main content form to refresh after closing the assets (images) popup to place syndicated (downloaded) images in the content.